apps/builder/app/env/env.server.ts

```diff
@@ -46,6 +46,10 @@ const env = {
   S3_BUCKET: process.env.S3_BUCKET,
   S3_ACL: process.env.S3_ACL,
   ASSET_CDN_URL: process.env.ASSET_CDN_URL,
+
+  // AI Features
+  OPENAI_KEY: process.env.OPENAI_KEY,
```

Comment:
when env is for plugins, we need to think of a way to be clear about that, maybe prefixing with plugin name, maybe passing it as a json string with plugin name as an env variable name?